airbyte-config-oss/init-oss/src/main/resources/seed/oss_catalog.json

```diff
@@ -15382,7 +15382,7 @@
     "sourceDefinitionId": "38f84314-fe6a-4257-97be-a8dcd942d693",
     "name": "Insightly",
     "dockerRepository": "airbyte/source-insightly",
-    "dockerImageTag": "0.1.1",
```

Comment:
Merging master in should fix these automated changes

Reply:
Oh actually I'm wrong, looks like they weren't updated correctly in the other PR. in any case I think these are valid chagnes

Reply:
I know a trick to get around this issue of unable to merge